app/docs/how-to/streamline-curbside.md

```diff
@@ -0,0 +1,255 @@
+* This URL will display all open lists on your Trello board. Choose the one you want and capture the `id` value. In this example, we're using the `Requests` list with an id of `5eb4674aed5ff17eae53760f`.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/trello-list-id.png" width="75%">
+</p>
+
+::: warning The board ID is unique to you!
+* The `DRzgczUF` value used in this example is a **unique** board identifier and the value for your Trello board **will be different**.
+:::
+
+
+### Putting Routegy and Trello together
+
+The final step is to connect Routegy and Trello using a [touch handler](/reference/touch-handlers/) that will create a new Trello card every time someone interacts with your touchpoint. 
+
+#### Step 1: Create a touch handler
+
+Go back to your Routegy workspace, select **Handlers** tab, and click **Create handler** button.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/handlers-tab.png" width="90%">
+</p>
+
+Give your new handler a name and description, and choose **Trello** from the **Handler Type** dropdown. Next, enter your **Trello API Key**, **API Token**, and **ListId** that you captured when configuring the Trello board [earlier](#finding-your-trello-api-key-and-token). 
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/handler-trello.png" width="75%">
+</p>
+
+Finally, scroll down to the **Card name** field, and replace its value with the following template: 
+```
+{{ name }} is waiting in {{ car }}, ordered via {{ order_type }}, arrived at {{ context.touch.created_at | date:"g:iA" }}
+```
+**Card name** field controls how Routegy will name Trello cards that it creates. The template above customizes the name so it will contain a complete information about arriving customers (e.g. *Pawel is waiting in Red Honda, ordered via Online, arrived at 5:42PM*)
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/handler-trello-card-name.png" width="75%">
+</p>
+
+Click **Create handler** to save all changes.
+
+#### Step 2: Attach touch handler to your touchpoint
+
+Go back to the **Touchpoints** tab and click on your curbside pickup touchpoint to open touchpoint details.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/workspace-edit-touchpoint.png" width="90%">
+</p>
+
+Click **Edit** to edit the touchpoint and add your Trello handler to it using the **Handler** field. Just click on the field, start typing you handler name, and then choose the handler from the list.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-add-handler.png" width="75%">
+</p>
+
+Click **Save changes** to save your touchpoint.
+
+#### Step 3: Test
+
+We everything configured, we are ready to test the solution, and make sure that it works as expected. Scan the QR code with your phone (or just click on it), fill the form just like your customer would, and watch a new Trello card being automatically created in your board.
```

Comment:
Now that everything is ready, let's test it out. Click the touchpoint link or scan the QR code with your phone, fill out the form, and watch a new Trello card appear automatically in your board.